datajunction-server/datajunction_server/construction/build.py

```diff
@@ -287,16 +291,49 @@ def _build_tables_on_select(
             Optional[ast.Table],
             _get_node_table(node, build_criteria),
         )  # got a materialization
+        fk_column_mapping = {
+            ",".join([pk.name for pk in col.dimension.current.primary_key()]): col
```

Comment:
This comma separated list of names... would it be safer to sort it by name or does the `primary_key()` method returns them in order?

Reply:
Good point, I'll add sorting by name. (Not that we support multi-column primary keys at the moment...)